Tasks/AzureResourceGroupDeployment/operations/ResourceGroup.ts

```diff
@@ -58,7 +58,7 @@ export class ResourceGroup {
         return new Promise<void>((resolve, reject) => {
             var extDelPromise = this.machineGroupExtensionHelper.deleteExtensionFromResourceGroup();
             var deleteRG = (val) => {
-                var armClient = new armResource.ResourceManagementClient(this.taskParameters.credentials, this.taskParameters.subscriptionId);
+                var armClient = new armResource.ResourceManagementClient(this.taskParameters.credentials, this.taskParameters.subscriptionId, { longRunningOperationRetryTimeout: this.taskParameters.timeout });
```

Comment:
Yes; I've added it in DeployRG model. So that I'll have to make very less changes later once we make that change on the server